tests/test_fields.py

```diff
@@ -169,15 +169,19 @@ class OtherSchema(MySchema):
         assert schema2.fields["foo"].key_field.root == schema2
         assert schema2.fields["foo"].value_field.root == schema2
 
+    # Regression test for https://github.com/marshmallow-code/marshmallow/issues/1357
     def test_datetime_list_inner_format(self, schema):
         class MySchema(Schema):
             foo = fields.List(fields.DateTime())
+            bar = fields.Tuple((fields.DateTime(),))
 
             class Meta:
                 datetimeformat = "iso8601"
+                dateformat = "iso8601"
```

Comment:
I don't think this has any effect since bar is a `DateTime` instance. Is `bar` supposed to be a `Date`?

Reply:
Ah crap, yes. Will fix